core/src/scene/styleParam.h

```diff
@@ -40,15 +42,16 @@ enum class StyleParamKey : uint8_t {
     text_source,
     text_wrap,
     transform,
+    transition_hide_time,
+    transition_selected_time,
+    transition_show_time,
     visible,
     width,
-    centroid,
-    collide,
-    transition_show_time,
-    transition_hide_time,
-    transition_selected_time
 };
 
+// UPDATE WITH StyleParamKey CHANGES!
+constexpr size_t StyleParamKeySize = static_cast<size_t>(StyleParamKey::width)+1;
```

Comment:
yeah.. even though it's just one line below the referenced value -- What about a last item for StyleParamKey::NUM_ELEMENTS to make it more obvious?